### DISCUSSION TITLE
r/city-china-zhejiang-jiaxing-haining

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#91] [Announcements]
# r/city-china-zhejiang-jiaxing-haining

嘉兴海宁，两天一晚旅行攻略

https://blog.ops-coffee.cn/r/city-china-zhejiang-jiaxing-haining.html

**Comment by qishu321** [comment]
大佬，现在跳去哪家公司啦？

**Comment by beyond1314** [comment]
看完了，写得好

**Comment by nomanfound** [comment]
第一个配图的是哪家店，感觉确实不错啊
